documentation/provider/powerdns.md

```diff
@@ -62,3 +62,14 @@ D("example.com", REG_NONE, DnsProvider(DSP_POWERDNS),
 
 ## Activation
 See the [PowerDNS documentation](https://doc.powerdns.com/authoritative/http-api/index.html) how the API can be enabled.
+
+## Caveats
+
+### SOA Records
+The SOA record is supported for use, but behavior is slightly different than expected.
+If the SOA record is used, [PowerDNS will not increase the serial](https://doc.powerdns.com/authoritative/dnsupdate.html#soa-serial-updates) if the SOA record content changes.
+This itself comes with exceptions as well, if the `SOA-EDIT-API` is changed to a different value the logic will update the serial to a new value.
+See [this issue for detailed testing](https://github.com/StackExchange/dnscontrol/pull/3404#issuecomment-2628989200) of behavior.
```

Comment:
Sorry for the late reply, I’ve been really busy. What’s the reasoning behind not including these results in the documentation? I believe it could be helpful for someone less familiar with GitHub. Also, with a code refactor, these examples could be updated.

cc: @veratil, @tlimoncelli.

Reply:
To me, the description of behavior is well enough defined without an overload of information for those that do not need it.